corehq/apps/repeaters/models.py

```diff
@@ -297,13 +297,13 @@ def handle_success(self, response, repeat_record):
         """handle a successful post
         """
         generator = self.get_payload_generator(self.format_or_default_format())
-        return generator.handle_success(response, self.payload_doc(repeat_record))
+        return generator.handle_success(response, self.payload_doc(repeat_record), repeat_record)
```

Comment:
Used later to add error message on repeat record. Added as argument for all method calls in code.